rewriter/SourceRewriter.cpp

```diff
@@ -130,6 +131,13 @@ static bool ignore_file(const Filename &filename) {
   return !filename.starts_with(OutputDirectory) && !filename.starts_with(RootDirectory);
 }
 
+static bool ignore_function(const Filename &filename, const Function &function) {
```

Comment:
Shouldn't be necessary with `--root-directory`.

Reply:
I want to specifically _include_ `ia2_compartment_destructor()`, which is outside the source directory, while still ignoring the rest of ia2.h and ia2_internal.h.